scripts/blacklist_onnx_chainer_tests.py

```diff
@@ -0,0 +1,17 @@
+import os
+import shutil
+
+
+BLACKLISTS = {
+    10: ['deconvolution2d_group3', 'deconvolutionnd_group3', 'resizeimages']
+}
+
+
+def main():
```

Comment:
Sorry I forgot to mention this, but to blacklist failing tests, we already have https://github.com/pfnet-research/chainer-compiler/blob/master/scripts/onnx_chainer_tests.py#L12